CHANGELOG.rst

```diff
@@ -15,3 +15,15 @@ CHANGELOG appropriately with each PR
 
 Release notes for previous/upcoming versions can be found in docs/source/releases,
 for reference.
+
+Installation Updates
```

Comment:
Should we just retag 1.10.0a3, so I will put these under v1_10_0a3 release notes?

Reply:
Yeah, that sounds fine to me.